### PR TITLE
Remove unused tx related methods from IStore (#1538)

### PR DESCRIPTION
**Pull request by sindydwns**
Issue address is #1538

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ To be released.
 
 ### Backward-incompatible API changes
 
+ -  Removed unused tx related methods from IStore [[#1538], [#2201]]
```
Comment (longfin) [inline CHANGES.md:22]:
```suggestion
 -  Removed unused transaction related methods from `IStore` and its
    implementations [[#1538], [#2201]]
```

**Review by longfin — DISMISSED** [review]
I left a small suggestion in the change history, but the rest are fine IMHO 🙇

**Review by OnedgeLee — APPROVED** [review]

File: CHANGES.md (hunk 1 above)
Comment (sindydwns) [reply CHANGES.md]: thanks. that makes sense.

**Review by longfin — APPROVED** [review]

**Comment by longfin** [comment]
@sindydwns Merged, thanks!

**Pull request merged**